docs/pages/setup/operations/ca-rotation.mdx

```diff
@@ -11,57 +11,79 @@ description: How to rotate Teleport's certificate authority
 (!docs/pages/includes/tctl.mdx!)
 
 <Admonition type="note">
-For cloud, login with a teleport user with editor privileges:
-```code
-# tsh logs you in and receives short-lived certificates
-$ tsh login --proxy=myinstance.teleport.sh --user=email@example.com
-# try out the connection
-$ tctl get nodes
-```
+  For cloud, login with a teleport user with editor privileges: ```code # tsh
```

Comment:
Looks like the reflow may have messed with the code snippet a bit here.

If we're editing this Admonition, I think we should go ahead and fix the capitalization of "cloud" and "teleport" and change "login" to "log in."